src/main/java/uk/gov/hmcts/reform/sscs/service/SubmitAppealServiceBase.java

```diff
@@ -76,7 +71,7 @@ public SubmitAppealServiceBase(CcdService ccdService,
         this.idamService = idamService;
         this.convertAIntoBService = convertAIntoBService;
         this.regionalProcessingCenterService = regionalProcessingCenterService;
-        this.caseAccessManagementFeature  = caseAccessManagementFeature;
+        this.caseAccessManagementFeature = caseAccessManagementFeature;
```

Comment:
don't think this feature flag is needed anymore. the feature is already enabled in production

Reply:
I didn't add this, it was added 15 days ago in this https://github.com/hmcts/sscs-tribunals-case-api/pull/3783 - my IDE just removed the double space on a reformat

Reply:
yes, I know. I just mentioned it as a cleanup we could do since we're updating the class. but it's not an issue